crates/modelardb_server/tests/integration_test.rs

```diff
@@ -602,6 +617,43 @@ fn test_can_create_register_and_list_multiple_tables_and_model_tables() {
     assert_eq!(retrieved_table_names, table_names);
 }
 
+#[test]
+fn test_can_drop_table() {
+    let mut test_context = TestContext::new();
+    test_context.create_table(TABLE_NAME, TableType::NormalTable);
+
+    test_context.drop_table(TABLE_NAME).unwrap();
+
+    let retrieved_table_names = test_context.retrieve_all_table_names().unwrap();
+    assert_eq!(retrieved_table_names.len(), 0);
+
+    // It should be possible to create a table, drop it, and then create a new table with the same name.
+    test_context.create_table(TABLE_NAME, TableType::NormalTable);
+}
+
+#[test]
+fn test_can_drop_model_table() {
+    let mut test_context = TestContext::new();
+    test_context.create_table(TABLE_NAME, TableType::ModelTable);
+
```

Comment:
Maybe check if table exist before dropping it to ensure the test does not succeed if the table is not created.

Reply:
Fixed.